### DISCUSSION TITLE
Which version solves compilation issue on illumos

### DISCUSSION THREAD

**Discussion by jcarnat** [ncruces/go-sqlite3#185] [General]
Hello,

I am facing the same issue than the one described [here](https://github.com/superseriousbusiness/gotosocial/issues/2952) while compiling GoToSocial 0.17.3 on illumos. As I understand their release note, this GtS version comes with go-sqlite3 v0.20.0.

Which is the go-sqlite3 version that solves PR 2952 ?

Thank you.

**Comment by ncruces** [comment]
That issue describes a problem building `modernc` SQLite, as a dependency of GoToSocial.

`ncruces/go-sqlite3` never had build issues that I know of, tests run in a OmniOS VM for every commit, and hasn't had major know bugs since [v0.18.1](https://github.com/ncruces/go-sqlite3/releases/tag/v0.18.1); [v0.20.0](https://github.com/ncruces/go-sqlite3/releases/tag/v0.20.0) improved the default configuration, if you're running in a (memory limited) VM.

The [support matrix page](https://github.com/ncruces/go-sqlite3/wiki/Support-matrix) describes the limitations (see the legend) and the [VFS design page](https://github.com/ncruces/go-sqlite3/blob/main/vfs/README.md) goes more in depth.

If you still have an issue, please explain what exactly is happening. For GoToSocial issues you'll have more luck using their issues board.

**Reply by jcarnat** [reply]
Indeed, compilation issues are happening in the context of a GoToSocial compilation. I couldn’t understand if this was due to GtS and/or modernc SQLite. But from your reply, I understand that my issue is probably more on GtS side. I’ll manage this with them.
Thank you.
